kubejs/client_scripts/base/jei_descriptions.js

```diff
@@ -357,6 +361,21 @@ JEIEvents.information((event) => {
                 ` `,
                 `For an easy supply, try an Aquatic Entangler with Junk Nets!`
             ]
+        },
+        {
+            items: [
+                "hexerei:sage_seed",
+                "immersiveengineering:seed",
+                "occultism:datura_seeds",
+                "sushigocrafting:cucumber_seeds",
+                "sushigocrafting:soy_seeds",
+                "sushigocrafting:wasabi_seeds",
+                "sushigocrafting:sesame_seeds"
+            ],
+            text: [
+                "This seed does not drop from grass!",
```

Comment:
I'm a little uncertain this is needed now. The Market properly shows in JEI, so it should come up immediately as an option for obtaining them. 

Dev already changed the text on the hexerei seed to remove their built in notification.

Reply:
If it stays, I think I'd rather see a shorter message: 'Purchase from the Market block'

Reply:
I've created this tooltip mostly because of feedback I've got from few people I was testing out this PR before Vanilla ore gen Boo made, So people for some reason are still getting confused that it doesn't drop from the grass even tho Market is visible in the JEI >.>
Gonna change it "Purchase from the Market block." tho as you asked 👍